src/ledger/index.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * an accounting reminder:
+ * https://en.wikipedia.org/wiki/Double-entry_bookkeeping
+ */
+
+import * as accounts from "./accounts"
+import * as queries from "./query"
+import * as transactions from "./transaction"
+
+type LoadLedgerOptions = {
```

Comment:
`LoadLedgerOptions` feels not accurate because it's not an option. maybe `LoadLedgerParams`?

Reply:
Renamed